### PR TITLE
Pathways V1

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**
- FAQ
- Profile images
- Call status
- Can cancel calls with a reason
- Biography for quotes
- sentry setup
- Call feedback for terminated

### CODE REVIEW

File: src/components/Dashboard/DashboardHeader.tsx [hunk 1]
```diff
@@ -2,12 +2,23 @@ import { Col, Space, Typography } from "antd";
...
-const DashboardHeader: React.FC = () => {
+interface Props {
+  openBio: (quote: Quote) => void;
+}
+
+const DashboardHeader: React.FC<Props> = ({ openBio }) => {
   const [dailyQuote] = useState(getRandomItem(QUOTES) as Quote);
+  const [background] = useState(getRandomItem(BACKGROUNDS));
   const [currTime, setCurrTime] = useState(new Date());
+  const [hasLiked, setHasLiked] = useState(false);
```
Comment (mpekala23) [inline src/components/Dashboard/DashboardHeader.tsx:21]: what is this liked feature going to do? are we going to eventually store it somewhere on the backend and expose it so people can see quotes they've liked in the past?

File: src/components/Modals/BiographyModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import BiographyModal from "./BiographyModal";
+
+export default BiographyModal;
```
Comment (mpekala23) [inline src/components/Modals/BiographyModal/index.tsx:3]: i know at one point we had been talking about removing these kind of index files in favor of just importing from the file. is there a reason we want to do it this way now (i noticed this in another spot in this PR as well)

File: src/components/Modals/CancelCallModal/CancelCallModal.tsx [hunk 3]
```diff
@@ -64,7 +77,14 @@ const CancelCallModal: React.FC<Props> = ({ data, closeModal }) => {
...
         placeholder={t("cancelCallModal.dropdownPlaceholder")}
-      ></Select>
+        onChange={(value: string) => setReason(value)}
+      >
+        {CANCEL_REASONS.map((reason) => (
+          <Select.Option key={reason} value={reason}>
+            {reason}
+          </Select.Option>
+        ))}
+      </Select>
```
Comment (mpekala23) [inline src/components/Modals/CancelCallModal/CancelCallModal.tsx:87]: 💯

File: src/redux/modules/call.ts [hunk 4]
```diff
@@ -200,7 +228,14 @@ export const callSlice = createSlice({
     builder.addCase(initializeRemotes.rejected, () =>
       showToast("initializeProducers", "Failed to initialize remotes.", "error")
     );
+    builder.addCase(cancelCall.fulfilled, (state, action) => {
+      callAdapter.updateOne(state, action.payload);
+      showToast("cancelCall", "Successfully cancelled call.", "success");
+    });
+    builder.addCase(cancelCall.rejected, () =>
+      showToast("cancelCall", "Failed to cancel call.", "error")
+    );
```
Comment (mpekala23) [inline src/redux/modules/call.ts:237]: Do we want to translate these `Successfully cancelled call.` and `Failed to cancel call.` strings?

File: src/redux/modules/session.ts [hunk 5]
```diff
@@ -109,6 +126,14 @@ export function sessionReducer(
...
         status: action.payload,
       };
+    case SET_PROFILE_IMAGE:
+      return {
+        ...state,
+        user: {
+          ...state.user,
+          profileImagePath: action.payload,
+        },
+      };
```
Comment (mpekala23) [inline src/redux/modules/session.ts:136]: do we want to transition this module to use `createSlice`?

**Review by mpekala23 — APPROVED** [review]
🎉 woot woot! looks great to me, a few questions / small ideas but nothing urgent

File: src/components/Call/WaitingRoomCard.tsx [hunk 6]
```diff
@@ -38,7 +38,7 @@ export const WaitingRoomCard = ({
   openInfoModal,
   openTestConnectionModal,
 }: Props) => {
-  const { t, i18n } = useTranslation("call");
+  const { t, i18n } = useTranslation(["call", "modals"]);
```
[the remark this answers is not in the document]
Comment (gabrielsaruhashi) [reply src/components/Call/WaitingRoomCard.tsx:41]: I have a .d.ts file!
it's typed

File: src/redux/modules/call.ts (hunk 4 above)
Comment (gabrielsaruhashi) [reply src/redux/modules/call.ts:237]: yep! good catch

File: src/redux/modules/session.ts (hunk 5 above)
Comment (gabrielsaruhashi) [reply src/redux/modules/session.ts:136]: ye! will open an issue for this

File: src/api/User.ts [hunk 7]
```diff
@@ -36,25 +41,46 @@ export async function loginWithCredentials(cred: {
...
+    await initializeSession(token, body.data, cred.language);
+  } catch (err) {
+    Store.dispatch(setSessionStatus("inactive"));
+    showToast("login_error", "Invalid ID or Pin Code", "error");
+  }
+}
+
+export async function updateProfile(profileImagePath: string) {
+  try {
+    await fetchAuthenticated("", {
```
[the remark this answers is not in the document]
Comment (gabrielsaruhashi) [reply src/api/User.ts:71]: nope! this is plain users/id

File: src/App.tsx [hunk 8]
```diff
@@ -54,12 +56,18 @@ function App({
     i18n.changeLanguage(session.language);
   }, [session.language, i18n]);
 
+  Sentry.init({
+    dsn: process.env.REACT_APP_SENTRY_DSN,
+    integrations: [new Integrations.BrowserTracing()],
```
[the remark this answers is not in the document]
Comment (gabrielsaruhashi) [reply src/App.tsx:61]: not that I know of lol

File: src/components/Modals/BiographyModal/index.tsx (hunk 2 above)
Comment (gabrielsaruhashi) [reply src/components/Modals/BiographyModal/index.tsx:3]: oh tony suggested we used this pattern as we're thinking about moving to Next js (SSR) 
it seems like indeed this is cleaner for some reason...
idk - we should ask him lol why this helps

File: src/components/Call/Call.tsx [hunk 9]
```diff
@@ -164,26 +177,52 @@ const CallBase: React.FC<Props> = React.memo(
...
+        "participantDisconnect",
+        async ({ id, type }: CallParticipant) => {
+          if (type === "user" && call?.userIds.includes(id)) {
+            playLeaveCall();
+            setParticipantsHasJoined(false);
+            openNotificationWithIcon(
+              `${call?.userParticipants[0].firstName} ${t(
+                "peer.joinedCallTitle"
+              )}.`,
+              t("peer.joinedCallBody"),
```
[the remark this answers is not in the document]
Comment (gabrielsaruhashi) [reply src/components/Call/Call.tsx:191]: great catch! tksss

File: src/components/Dashboard/DashboardHeader.tsx (hunk 1 above)
Comment (gabrielsaruhashi) [reply src/components/Dashboard/DashboardHeader.tsx:21]: we'll use segment to collect this data!

**Pull request merged**